src/response.ts

```diff
@@ -32,6 +32,7 @@ const commonSnapshotResponse = (snapshot, blocksProperty) => ({
 
 export const globalSnapshotResponse = (snapshot) => ({
   ...commonSnapshotResponse(snapshot, "dag_blocks"),
+  epochProgress: snapshot.epoch_progress,
```

Comment:
Should this go into the commonSnapshotResponse? It's part of the core snapshot representation.